harper-dictionary-parsing/src/char_string.rs

```diff
@@ -16,4 +17,7 @@ impl CharStringExt for [char] {
 
         out
     }
+    fn to_string(&self) -> String {
```

Comment:
Would it be possible to implement `Display` instead?

Reply:
Only traits defined in the crate can be implemented on types outside of the crate, so I don't think so, no.